.env.dist

```diff
@@ -6,3 +6,8 @@ EMAIL_HOST_USER=
 EMAIL_USE_TLS=
 EMAIL_HOST_PASSWORD=
 GCS_SERVICE_ACCOUNT_KEY=
+WORKSPACES_DATABASE_ROLE=
```

Comment:
Is this needed? All the values can be in the `docker-compose.yaml` file

Reply:
Not really needed.